Modules/mathmodule.c

```diff
@@ -3386,13 +3388,15 @@ perm_comb_small(unsigned long long n, unsigned long long k, int iscomb)
             return PyLong_FromUnsignedLongLong(comb_odd_part << shift);
         }
 
-        /* Only contains items larger than in fast_comb_limits1. */
-        /* long long is at least 64 bit. */
+        /* Maps k to the maximal n so that 2*k-1 <= n <= 127 and C(n, k)*k
+         * fits into a long long (which is at least 64 bit).  Only contains
+         * items larger than in fast_comb_limits1. */
         static const unsigned long long fast_comb_limits2[] = {
```

Comment:
> which is at least 64 bit

C99 provides [types like `uint_least64_t`](https://en.cppreference.com/w/c/types/integer) that express such comments explicitly. So I think it will be better to `#include <stdint.h>` and use it:

```suggestion
        /* Maps k to the maximal n so that 2*k-1 <= n <= 127 and C(n, k)*k
         * fits into a long long.  Only contains
         * items larger than in fast_comb_limits1. */
        static const uint_least64_t fast_comb_limits2[] = {
```

Reply:
It should contain at least LLONG_MAX. There is no guarantee that `uint_least64_t` is at least so large as positive `long long`.